jdbc/src/main/scala/zio/sql/jdbc.scala

```diff
@@ -37,4 +44,7 @@ trait Jdbc extends zio.sql.Sql with TransactionModule with JdbcInternalModule wi
 
   def execute(update: Update[_]): ZIO[SqlDriver, Exception, Int] =
```

Comment:
We may want to rename this to make clearer that it does not run as part of a transaction